docs/scos/dev/guidelines/performance-guidelines/spryker-elastic-computing/enable-queue-task-flow-in-new-relic.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Enable queue task flow in New Relic
```

Comment:
What do you mean by `queue task flow`? The doc seems to explain something different

Reply:
In general this references to the queue processing in P&S infrastructure.